tmdrv_devices/thrustmaster_tmx.py

```diff
@@ -0,0 +1,9 @@
+name = 'Thrustmaster TMX'
+idVendor = 0x044f
+idProduct = [0xb67e, 0xb65d, 0xb67f]
+control = [
+	{'step':1, 'request_type':0x41, 'request':83, 'value':0x0001, 'index':0x0000, 'data':b''},
+	{'step':2, 'request_type':0x41, 'request':83, 'value':0x0007, 'index':0x0000, 'data':b''},
+]
+jscal = '6,1,255,32767,32767,21844,21844,1,3,511,511,1394469,1394469,1,3,511,511,1394469,1394469,1,3,511,511,1394469,1394469,1,0,0,0,536870912,536870912,1,0,0,0,536870912,536870912'
+dev_by_id = 'usb-Thrustmaster_Thrustmaster_TMX_Racing_Wheel-joystick'
```

Comment:
It looks as though the dev_by_id is correct. You'll still need to adjust the jscal data, after following the instructions I left earlier to calibrate the device. If you need assistance with this, feel free to ask questions.